cli/test/test_setup_wif.py

```diff
@@ -0,0 +1,152 @@
+import sys
+import requests
+import os
+import string
+import random
+import unittest
+from dotenv import load_dotenv
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
```

Comment:
Why do we need this? Can you add a comment here please?

Reply:
moved test to same dir as a script and now it is not necessary